components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```diff
@@ -53,30 +53,38 @@ def main(argv):
     try:
         # Validate and load config file.
         config_file_path = pathlib.Path(parsed_args.config)
-        clp_config = load_config_file(config_file_path, default_config_file_path, clp_home)
+        clp_config = load_config_file(
```

Comment:
Inside `load_config_file`, we print one of the arguments in a log, which I think is wrong, right? I'm fine leaving it as is and opening an issue.

Reply:
Right, the log would be confusing to general users when `load_config_file` is used in the "bootstrap" scripts. I didn't modify `load_config_file` directly because the "native" scripts also uses `load_config_file` - it's possible to use a flag to determine whether the method is called in a bootstrap script or a native script though.

@coderabbitai please create an issue for the inaccurate log and we will fix it as we clean up the bootstrap / native scripts in a future PR